packages/prisma/schema.prisma

```diff
@@ -302,6 +302,7 @@ model User {
   movedToProfile             Profile?                     @relation("moved_to_profile", fields: [movedToProfileId], references: [id], onDelete: SetNull)
   secondaryEmails            SecondaryEmail[]
   isPlatformManaged          Boolean                      @default(false)
+  showSupport                Boolean                      @default(true)
```

Comment:
we dont need to save this on a db level. we should do this either localStorage or as a cookie